data/tribes/initialization/empire/units.lua

```diff
@@ -1246,8 +1246,18 @@ wl.Descriptions():new_tribe {
             lore_author = pgettext("empire_building", "An enthusiastic stonemason arriving to a new quarry"),
             -- TRANSLATORS: Purpose helptext for an Empire production site: Quarry
             purpose = pgettext("empire_building", "Cuts blocks of granite and marble out of rocks in the vicinity."),
-            -- TRANSLATORS: Note helptext for an Empire production site: Quarry
-            note = pgettext("empire_building", "The quarry needs rocks to cut within the work area.")
+            note = {
+               -- TRANSLATORS: Note helptext for an Empire production site: Quarry, part 1
+               pgettext("empire_building", "The quarry needs rocks to cut within the work area."),
+               -- TRANSLATORS: Note helptext for an Empire production site: Quarry, part 2
+               pgettext("empire_building", "The stonemason finds 5 blocks of granite and 2 blocks of marble out of 7 rocks.")
+            },
+            performance = {
+               -- TRANSLATORS: Performance helptext for an Empire production site: Quarry, part 1
+               pgettext("empire_building", "This building can produce one granite or one marble between %1$s and %2$s, depending on how far the stonemason has stones."):bformat(format_seconds(42), format_minutes_seconds(1, 7)),
```

Comment:
"one block of stone"?

Shouldn't we give the values for granite and marble separately? (59s-1m34s, 2m28s-3m56s)

Reply:
In the previous PR, when a full cycle had more of some items than of others, we always gave the averages separately. This would be harder for users to calculate for themselves.

Reply:
well I believe giving the time of one walk might be easier to understand, but for consistency we can do it like you suggested.